tests/virt_autotest/virt_utils.pm

```diff
@@ -30,19 +29,23 @@ our @EXPORT
 
 sub enable_debug_logging {
 
-    #turn on debug and log filter for libvirtd
-    #set log_level = 1 'debug'
-    #the size of libvirtd with debug level and without any filter on sles15sp3 xen is over 100G,
-    #which consumes all the disk space. Now get comfirmation from virt developers,
-    #log filter is set to store component logs with different levels.
-    my $libvirtd_conf_file = "/etc/libvirt/libvirtd.conf";
-    if (!script_run "ls $libvirtd_conf_file") {
-        script_run "sed -i '/^[# ]*log_level *=/{h;s/^[# ]*log_level *= *[0-9].*\$/log_level = 1/};\${x;/^\$/{s//log_level = 1/;H};x}' $libvirtd_conf_file";
-        script_run "sed -i '/^[# ]*log_outputs *=/{h;s%^[# ]*log_outputs *=.*[0-9].*\$%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%};\${x;/^\$/{s%%log_outputs=\"1:file:/var/log/libvirt/libvirtd.log\"%;H};x}' $libvirtd_conf_file";
-        script_run "sed -i '/^[# ]*log_filters *=/{h;s%^[# ]*log_filters *=.*[0-9].*\$%log_filters=\"1:qemu 1:libvirt 4:object 4:json 4:event 3:util 1:util.pci\"%};\${x;/^\$/{s%%log_filters=\"1:qemu 1:libvirt 4:object 4:json 4:event 3:util 1:util.pci\"%;H};x}' $libvirtd_conf_file";
-        script_run "grep -e log_level -e log_outputs -e log_filters $libvirtd_conf_file";
+    # libvirtd is only present in SLE serials product(SLEM and ALP)
+    # it has been replaced with modular daemons in Tumbleweed since libvirt 9.1.0
+    unless (is_libvirt91) {
```

Comment:
Hi Julie, I see nothing done for libvirt 9.1+ for debug enablement. Nothing needs to be done?

Reply:
I saw debugging enablement is called in unified_guest_installation.pm, so I skip it in autoyast profiles.

Reply:
Oh, I get your meaning. The services have changed since libvirt 9.1, original configuration files have not taken effects any longer. I'll find new configurations in the future after understanding more about libvirt.